packages/zk-transfer-web/src/libs/types.ts

```diff
@@ -0,0 +1,60 @@
+export type ERCToken = Map<any, any>;
+
+export interface Note extends Map<string, any> {
```

Comment:
Interesting. Did you have to generate this list by hand?
If so, that seems very labor-intensive, and I'm sorry you had to go through that.
If it appears in the future that you'll have to do something very repetitive like this, check with me first.

Given your experience here, you might be able to help me simplify or re-write a different part of the project, called a typed transform pipeline.

Reply:
I did generate it by hand. Just to stop the errors from typescript.

Reply:
Thanks for doing that this time!